python-threatexchange/threatexchange/extensions/video_vpdq/video_vpdq.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Wrapper around the vpdq signal type.
+"""
+
+import vpdq  # type: ignore
+from .vpdq_util import json_to_vpdq, vpdq_to_json
+from .vpdq_brute_hasher import match_VPDQ_hash_brute
+import pathlib
+import typing as t
+from threatexchange.content_type.content_base import ContentType
+from threatexchange.content_type.video import VideoContent
+import re
+from threatexchange.signal_type import signal_base
+
+
+class VideoVPDQSignal(signal_base.SimpleSignalType, signal_base.TextHasher):
+    """
+    Simple signal type for video using VPDQ.
+    Read about VPDQ at https://github.com/facebook/ThreatExchange/tree/main/vpdq
+    """
+
+    INDICATOR_TYPE = "HASH_VIDEO_VPDQ"
+    VPDQ_CONFIDENT_DISTANCE_THRESHOLD = 31
+    VPDQ_CONFIDENT_QUALITY_THRESHOLD = 50
+
+    @classmethod
+    def get_content_types(self) -> t.List[t.Type[ContentType]]:
+        return [VideoContent]
+
+    @classmethod
+    def validate_signal_str(cls, signal_str: str) -> str:
+        """VPDQ hex str contains 64 hexidecimal characters."""
```

Comment:
blocking: Isn't it a series of hashes? With quality scores, frame numbers, and timestamps?